chart/templates/workspace-www/configmap.yaml

```diff
@@ -12,13 +12,17 @@ metadata:
   annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
   {{- end }}
 data:
+  {{- if and (ne .Values.appConfigValues.bigqueryOauth2ClientId "") (ne .Values.appSecrets.bigqueryOauth2ClientSecret.value "") }}
```

Comment:
Ey, this is not well thought. Secret could be passed without the "value" field (remember that you can specify it from an external secret)

To simplify it, you can limit the condition to only take into account `Values.appConfigValues.bigqueryOauth2ClientId` that is going to simplify a lot and the result is going to be the same.

TIP. Check how I check if a variable is empty: https://github.com/CartoDB/carto-selfhosted-helm/pull/201/files#diff-8fd8e35b22894402e4888a546d0e531e3e36caae91af385c8eb0d42e3b80c7efR5